src/styles/main.css

```diff
@@ -0,0 +1,423 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  margin: 0 auto;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  width: 727px;
+  gap: 1px;
+  padding: 10px;
+}
+
+.calendar__day {
+  position: relative;
+  box-sizing: border-box;
+  display: flex;
+  flex: 0 0 100px;
+  height: 100px;
+  background-color: #eee;
+  border: 1px solid black;
+}
+
+.calendar__day--1::before {
+  content: '1';
+  display: flex;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  color: black;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day--2::before {
+  content: '2';
+  display: flex;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  color: black;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day--3::before {
+  content: '3';
+  display: flex;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  color: black;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day--4::before {
+  content: '4';
+  display: flex;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  color: black;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day--5::before {
+  content: '5';
+  display: flex;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  color: black;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day--6::before {
+  content: '6';
+  display: flex;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  color: black;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day--7::before {
+  content: '7';
+  display: flex;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  color: black;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
```

Comment:
The styles for `.calendar__day--1::before` to `.calendar__day--7::before` are repetitive. Consider using a single class for these styles and dynamically setting the content property using a data attribute or a similar approach to avoid repetition.